Podfile

```diff
@@ -1,8 +1,7 @@
-platform :osx, '10.9'
+platform :osx, '10.12'
```

Comment:
(If that were the case our podspec file would be wrong so I guess that's not the reason)